code/style.css

```diff
@@ -1,7 +1,333 @@
+  font-size: 40px;
+  opacity: 0.6;
+  color: #fdffb6;
+}
+
+.video-container {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  overflow: hidden;
+  z-index: -1;
+}
+
+video {
+  object-fit: fill;
+  width: 100%;
+  height: 100vh;
+  z-index: 0;
+}
+
+picture {
+  display: none;
+}
+
+nav {
+  position: absolute;
+  width: 50%;
+  top: 0;
+  background-color: transparent;
+  margin: 10px;
+  overflow: hidden;
+}
+
+nav a {
+  text-decoration: none;
+  font-size: 18px;
+  letter-spacing: 1px;
+  box-sizing: border-box;
+  display: inline-block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 16px 24px;
+}
+/***sign in button***/
+.sign-in {
+  position: absolute;
+  font-size: 20px;
+  display: inline-block;
+  transition: all 0.4s ease 0s;
+  border: 3px solid white;
+  background-color: transparent;
+  color: white;
+  margin-top: 20px;
+  margin-left: 500px;
+}
+.sign-in:hover {
+  background: #fdffb6;
+  color: grey;
+}
+
+.sign-in:active {
+  background: #fdffb6;
+  color: grey;
+}
+
+/***header for tablet/mobile***/
+@media screen and (max-width: 1020px) {
+  div.header-text h1 {
+    top: 40%;
+  }
+  .header-text span {
+    top: 50%;
+    font-size: 20px;
+  }
+  .video-container {
+    display: none;
+  }
+  .menu {
+    display: none;
+  }
+  .sign-in {
+    display: none;
+  }
+  picture {
+    width: 100%;
+    height: auto;
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: inline-block;
+    position: absolute;
+    margin: 0;
+    padding: 0;
+    top: 0;
+    left: 0;
+    overflow: hidden;
+    z-index: -1;
+  }
+}
+
+/***sign up form***/
+.main {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.signup {
+  flex: 1;
+  margin-right: 20px;
+}
+.accordion {
+  flex: 1;
+}
+
+form {
+  max-width: 50vw;
+  margin-left: 15px;
+}
+
+legend {
+  font-family: "Lexend Giga", sans-serif;
+  color: grey;
+  text-decoration: none;
+  font-size: 20px;
+  border-bottom: 0px !important;
+  margin-left: 15px;
+}
+
+.form-group {
+  color: white;
+  font-family: "Lexend Giga", sans-serif;
+  font-weight: 200;
+  font-size: 20px;
+  border-radius: 5px;
+  line-height: 30px;
+  background-color: transparent;
+  border: 2px solid #fdffb6;
+  transition: all 0.5s;
+  padding: 15px;
+  margin-bottom: 15px;
+  width: 50vw;
+  box-sizing: border-box;
+  outline: 0;
+  color: grey;
+}
+
+.form-group:focus {
+  border: 2px solid #fcff99;
+}
+
+.selection {
+  background-color: lightgrey;
+  color: grey;
+  font-family: "Lexend Giga", sans-serif;
+}
+select {
+  background: lightgrey;
+}
+
+.submit {
+  font-family: "Crimson", Arial, Helvetica, sans-serif;
+  width: 50vw;
+  background: #fcff99;
+  border-radius: 5px;
+  border: 0;
+  cursor: pointer;
+  color: #908aa8;
+  font-size: 24px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  transition: all 0.3s;
+  margin-top: 5px;
+  font-weight: 700;
+}
+.submit:hover {
+  background: #ececd3;
+}
```

Comment:
Really cool with combining the transition with the hover effect, keep exploring with the styling features 💥